TheOracle2/Clock/CampaignClock.cs

```diff
@@ -11,7 +11,7 @@ public CampaignClock(ClockSize segments, int filledSegments, string title, strin
     }
 
     public override string EmbedCategory => "Campaign Clock";
-    public override string LogMessage => "The event is triggered or the project is complete. Envision the outcome and the impact on your setting.";
+    public override string ClockFillMessage => "The event is triggered or the project is complete. Envision the outcome and the impact on your setting.";
```

Comment:
some widgets can produce more than one kind of alert output (scene challenges for instance, which are both a clock and a progress track), so a single LogMessage won't work for them